frontend/src/libs/components/modal/steps/delete-account/libs/types/delete-account-form-payload.ts

```diff
@@ -0,0 +1,11 @@
+type DeleteAccountFormPayload = {
+  describeYourSituation: string;
+  checkboxes: {
+    checkbox1: boolean;
+    checkbox2: boolean;
+    checkbox3: boolean;
+    checkbox4: boolean;
```

Comment:
This approach is difficult to extend, because for any new reason option we need to create new checkbox prop in dto. Please check how mupltiple checkboxes were handled in survey page [here](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/89/files#diff-92edabe5fdb8ef427ad887499a7713ef8d3a1a13638ab7bbf1780af60806318b) on the line 41, they all have the same name and are validated as [array of strings](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/89/files#diff-c89514f464530a1aae8beb2bd048c37ab23832896c9f2d58e85fafc55f6d958d)